spec/features/lateness_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
```

Comment:
It tests that you end up back on the homepage after doing whatever it was the individual features say.

Like I mentioned the other day, I'm not sure how I feel about the lateness task (unlike the other two tasks) not having a "check your answers" page – the features for the cost determination task feel nicer because at the end you can simply say "expect cost to be £x".